services/sendMail.js

```diff
@@ -1,17 +1,39 @@
 const nodemailer = require('nodemailer');
 
-const transporter = nodemailer.createTransport({
-    host: process.env.EMAIL_HOST,
-    path: process.env.EMAIL_PATH,
-    auth: {
-        user: process.env.EMAIL_ID,
-        pass: process.env.EMAIL_PASSWORD
+function createTransporter(context) {
+    let emailId, emailPassword;
+    
+    switch (context) {
+    case 'welcome':
+        emailId = process.env.WILKOMMEN_EMAIL_ID;
```

Comment:
I would suggest to keep the same keys itself for this basic email. Not necessary to update it to add willommen email